database/youtube/machine-learning.json

```diff
@@ -1,8 +1,18 @@
-[{
-    "name": "3Blue1Brown",
-    "description": "3Blue1Brown, by Grant Sanderson, is some combination of math and entertainment, depending on your disposition. The goal is for explanations to be driven by animations and for difficult problems to be made simple with changes in perspective.",
-    "url": "https://www.youtube.com/@3blue1brown",
-    "category": "youtube",
-    "subcategory": "machine-learning",
-    "language": "english"
-}]
+[
+    {
+        "name":"Udacity",
+        "description":"Free online course for learning tensorflow for Deep learning",
+        "url":"https://learn.udacity.com/courses/ud187",
+        "category":"online platform",
+        "subcategory":"tensorflow",
+        "language":"english"
+    },
+    {
+        "name": "3Blue1Brown",
+        "description": "3Blue1Brown, by Grant Sanderson, is some combination of math and entertainment, depending on your disposition. The goal is for explanations to be driven by animations and for difficult problems to be made simple with changes in perspective.",
```

Comment:
```suggestion
        "description": "3Blue1Brown, by Grant Sanderson, is a combination of math and entertainment, depending on your disposition. The goal is for difficult problems to be made simple with changes in perspective.",
```

Reply:
Thanks for review, I will make those changes.